app/grandchallenge/products/templates/products/partials/product_image_carousel.html

```diff
@@ -25,7 +25,9 @@
   </div>
   <div class="text-muted small"> Source: {{product.company.website}}</div>
 
-{% elif product.product_img %}
+{% elif product.product_img %} <!--TODO: elif len(product.images.all)==1, no carousel-->
```

Comment:
Nitpick: it's nicer to use `{# my comment here #}` rather than `<!-- my comment here -->` in Django.

Reply:
fixed, good to know ;)